lib/aur-sync

```diff
@@ -264,7 +264,7 @@ if type -P >/dev/null vifm; then
     viewer() ( cd "$1"; vifm -c 'view!' -c '0' - )
 else
     # shellcheck disable=SC2086
-    viewer() ( cd "$1"; command -- ${PAGER:-less -K} )
+    viewer() ( cd "$1"; command -- ${PAGER:-less -K -+F} )
```

Comment:
@rafasc is `-+F` a typo, should it be just `-F`? I believe `+F` means "follow", i.e. never exit, while `-F` is indeed exit if it fits on one screen.

Reply:
No, it's not a typo. The idea is that we *do not* want to quit if one screen, even if the user has `LESS=F`. The `-+` is a mechanism to override options set by the LESS var.

excerpt from `man less` for convenience:
> The environment variable is parsed before the command line, so command line options override the LESS environment variable.  If an option appears in
>        the LESS variable, it can be reset to its default value on the command line by beginning the command line option with "-+".

Reply:
Wow, I see, thanks :)

Reply:
Thank you for keeping an eye on things. :)